config/README.md

```diff
@@ -100,6 +100,11 @@ There are 4 files you must provide:
    resource][chrMap] provides such files for various organisms, and in the
    expected format.
 
+5. **OPTIONAL**: A **BED6** file with regions for which to produce
+   [ASCII-style alignment pileups][ascii-pileups] for. If not provided, no
+   pileups will be generated. Refer [here][bed-format] to see the expected
+   format.
```

Comment:
As above. But I'm wondering: Why do we have the exact same docs in two different places? :thinking:

Reply:
As I mentioned in a comment above:



>It is not 100% copied, but part of it. This README is required by the 
>Snakemake Catalogue. Only a few things should be modified there - if 
>necessary - in the future. What should I do with it?
>
>(This README only contains how to/set up things to run the workflow, and 
>the expected inputs and outputs)

Reply:
Ah, sorry, didn't catch that. Fine then.